### PR TITLE
Parent child hierarchy model

### PR DESCRIPTION
**Pull request by Chandrasekar-Rajasekar**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java [hunk 1]
```diff
@@ -113,19 +77,24 @@
             "    population BIGINT DEFAULT NULL,\n" +
             "    PRIMARY KEY (id),\n" +
             ");\n" +
-            "CREATE INDEX countryLongitudeIdx ON country (longitude);\n " +
-            "CREATE INDEX countryLatitudeIdx ON country (latitude);\n " +
-            "CREATE INDEX countryWikiIdx ON country (wikiId);\n" +
-            "CREATE INDEX countryLabelIdx ON country (label);\n ";
+            "CREATE INDEX placeLongitudeIdx ON place (longitude);\n " +
```
Comment (aklish) [inline db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java]: Remove index on lat/long for places.

File: db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java [hunk 2]
```diff
@@ -113,19 +77,24 @@
...
+            "CREATE INDEX placeLatitudeIdx ON place (latitude);\n " +
+            "CREATE INDEX placeWikiIdx ON place (wikiId);\n" +
+            "CREATE INDEX placeLabelIdx ON place (label);\n ";
+
+    private static final String CREATE_RELATIONSHIPS = "CREATE TABLE IF NOT EXISTS relationship_hierarchy\n" +
+            "(\n" +
+            "    childId VARCHAR(255) DEFAULT NOT NULL,\n" +
+            "    parentId VARCHAR(255) DEFAULT NULL,\n" +
+            "    PRIMARY KEY (childId, parentId),\n" +
```
Comment (aklish) [inline db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java]: Will this create an individual index for childId and an individual index for parentId - or just an index on the composite key.  We really need the former.

Comment (Chandrasekar-Rajasekar) [reply db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java]:
```Index Name = PRIMARY_KEY_CA; Column Name = ID; Table Name = HEALTH_RECORDS
Index Name = HEALTHRECORDSREGIONIDIDX; Column Name = REGIONID; Table Name = HEALTH_RECORDS
Index Name = HEALTHRECORDSDATEIDIDX; Column Name = REFERENCEDATE; Table Name = HEALTH_RECORDS
Index Name = PRIMARY_KEY_5; Column Name = ID; Table Name = LATEST_HEALTH_RECORDS
Index Name = LATESTHEALTHRECORDSREGIONIDIDX; Column Name = REGIONID; Table Name = LATEST_HEALTH_RECORDS
Index Name = PRIMARY_KEY_C; Column Name = CHILDID; Table Name = RELATIONSHIP_HIERARCHY
Index Name = PRIMARY_KEY_C; Column Name = PARENTID; Table Name = RELATIONSHIP_HIERARCHY
Index Name = PRIMARY_KEY_4; Column Name = ID; Table Name = PLACE
Index Name = PLACEWIKIIDX; Column Name = WIKIID; Table Name = PLACE
Index Name = PLACELABELIDX; Column Name = LABEL; Table Name = PLACE```

Comment (Chandrasekar-Rajasekar) [reply db-builder/src/main/java/com/yahoo/covid19/database/DatabaseBuilder.java]: We have two separate index. But in same name.`PRIMARY_KEY_C`

**Review by aklish — APPROVED** [review]

**Pull request merged**